includes/admin/OrderSettings.php

```diff
@@ -250,7 +250,7 @@ private function setAgeCheck(): void
         $ageCheckOfProduct           = $this->getAgeCheckOfProduct();
         $ageCheckFromSettings        = (bool) WCMYPA()->setting_collection->getByName($settingName);
 
-        $this->ageCheck = $ageCheckFromShipmentOptions ?? $ageCheckOfProduct ?? $ageCheckFromSettings;
```

Comment:
Nu gaat hij, als je age check zelf uitgezet hebt in de shipment options dit weer laten overschrijven door de product en settings variant van de optie, waardoor hij uiteindelijk toch weer `true` wordt. Dit is toch niet de bedoeling?

Reply:
Done